doc/api/child_process.md

```diff
@@ -43,6 +43,10 @@ the first one case-insensitively matching `PATH` to perform command lookup.
 This may lead to issues on Windows when passing objects to `env` option that
 have multiple variants of `PATH` variable.
 
+On Windows Node.js will sanitize the `env` by removing case-insensitive
+duplicates. Only first (in lexicographic order) entry will be passed to the
+child process.
```

Comment:
In JS object syntax, including object spread, it’s the *last* key that wins, so I it might be a bit more intuitive to use that?

Reply:
It chooses the first one for `PATH` (the doc about this is one line above this change), so I've kept that.

Reply:
IIRC the PATH deduplication was done in libuv. Should we be doing the deduplication all in one place?

Reply:
lexicographic order is correct in order to match the expectation of platform APIs like GetEnvironmentVariable which stops at the first lexicographic match

Reply:
@richardlau I expect this change to break stuff. It probably should be semver-major, which is kinda impossible to do in libuv.